build.gradle.kts

```diff
@@ -85,39 +85,6 @@ subprojects {
 
     tasks.withType<Test> {
         maxHeapSize = "8192m"
-        environment = mapOf(
-            "CORS_HOSTS" to "",
-            "NEWM_JWT_SECRET" to "",
-            "DATABASE_JDBC_URL" to "",
-            "DATABASE_USERNAME" to "",
-            "DATABASE_PASSWORD" to "",
-            "GOOGLE_CLIENT_ID" to "",
-            "GOOGLE_CLIENT_SECRET" to "",
-            "FACEBOOK_CLIENT_ID" to "",
-            "FACEBOOK_CLIENT_SECRET" to "",
-            "LINKEDIN_CLIENT_ID" to "",
-            "LINKEDIN_CLIENT_SECRET" to "",
-            "EMAIL_AUTH_SMTP_HOST" to "",
-            "EMAIL_AUTH_SMTP_PORT" to "",
-            "EMAIL_AUTH_USERNAME" to "",
-            "EMAIL_AUTH_PASSWORD" to "",
-            "EMAIL_AUTH_FROM" to "",
-            "SENTRY_DSN" to "",
-            "CLOUDINARY_URL" to "",
-            "AWS_REGION" to "",
-            "AWS_AUDIO_BUCKET" to "",
-            "AWS_AUDIO_SQS_QUEUE_URL" to "",
-            "AWS_AUDIO_CLOUDFRONT_HOST_URL" to "",
-            "AWS_AGREEMENT_BUCKET" to "",
-            "AWS_KMS_KEY_ID" to "",
-            "AWS_MINTING_SQS_QUEUE_URL" to "",
-            "IDENFY_API_KEY" to "",
-            "IDENFY_API_SECRET" to "",
-            "IDENFY_SIGNATURE_KEY" to "",
-            "NEWM_CHAIN_HOST" to "",
-            "NEWM_CHAIN_PORT" to "",
-            "NEWM_CHAIN_JWT" to "",
-        )
```

Comment:
Found out by accident that this is no longer needed for our unit tests - probably after one of the Ktor updates